web-registry/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -34,8 +38,21 @@ const sxs = {
 };
 
 const Dashboard = (): JSX.Element => {
+  const { wallet } = useWallet();
   const theme = useTheme();
   const isIssuer = useQueryIfIssuer();
+  const isCreditClassCreator = useQueryIfCreditClassCreator();
+  const classes = useQueryListClasses();
+  const adminClasses = classes?.classes.filter(
```

Comment:
i guess eventually this may need to be passed into the `MyCreditClasses`. since we will want to render these there. (see [here](https://github.com/regen-network/regen-web/pull/1118/files#r940544823))